### PR TITLE
CONTRIBUTORS file for Chapel 1.21

### PR DESCRIPTION
**Pull request by bradcray**
This is the CONTRIBUTORS.md file for Chapel 1.21.  Thanks to everyone who contributed!

### CODE REVIEW

**Comment by bradcray** [comment]
@chapel-lang/core-contributors: Please make sure that anyone whose PRs you helped review has received

@ankingcodes, @Aniket21mathur, @Rapiz1, @akshansh2000, @jhoole, @immadisairaj, @OmarElawady , @krishnadey30, @soohoonchoi, @rahulghangas, @klakho0400 @sriramsk1999 , @rajshekhar7, @wongalvis14, @dgarvit: Please make sure I've spelled your names correctly and alphabetized by the appropriate one.

Thanks!

**Comment by Aniket21mathur** [comment]
@bradcray yes it is  correct. Thanks!

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -7,35 +7,35 @@ Chapel release:
...
-* Sanket Chaudhari, individual contributor
-* Anway De, individual contributor
+* Soohoon Choi, [HPE]
 * Garvit Dewan, [GSoC 2019] student from [Indian Institute of Technology Roorkee]
 * Krishna Kumar Dey, [GSoC 2019] student from [Indian Institute of Information Technology, Sri City] (former individual contributor)
 * Lydia Duncan, [HPE]
+* Omar Elawady, individual contributor
 * Michael Ferguson, [HPE]
+* Rahul Ghangas, individual contributor
 * Ben Harshbarger, [HPE]
```
Comment (ben-albrecht) [inline CONTRIBUTORS.md]: @bradcray - merged a contribution from @SamuelHoward last week:


```suggestion
* Ben Harshbarger, [HPE]
* Samuel Howard,  individual contributor
```

Comment (bradcray) [reply CONTRIBUTORS.md]: Got it, thanks!  (and thanks @SamuelHoward!)

Comment (SamuelHoward) [reply CONTRIBUTORS.md]: My pleasure!

**Pull request merged**